backend/worker/quivr_worker/celery_worker.py

```diff
@@ -129,8 +176,12 @@ async def process_crawl_task(
     logger.info(
         f"Task process_crawl_task started for url={crawl_website_url}, knowledge_id={knowledge_id}, brain_id={brain_id}, notification_id={notification_id}"
     )
-
-    brain_vector_service = BrainVectorService(brain_id)
+    global engine
+    assert engine
+    with Session(engine, expire_on_commit=False, autoflush=False) as session:
```

Comment:
Is there any reason why this session is sync and not async ?

Reply:
because vector service is sync